### DISCUSSION TITLE
Screenshot initiation thru compass short tap

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#569] [Q&A] [answered]
Of course, I have been using the option to spontaneously save favorites during a tour ever since the function was introduced. However, every now and then I also want to take a screenshot. For this today I've to use the hardware function on my smartphone. I would like to use the same procedure as for "Add favorite". Could the panel on the short tap to the compass button be extended:

<img height="200" src="https://github.com/devemux86/cruiser/assets/121806238/622a35ac-ccb2-4f27-bbd2-3e4e209f236e"/>

Would be great!

**Comment by devemux86** [comment]
@WolfgangZ42 thanks for the suggestion.

We could do this.

An obstacle is that file operations in new Androids are now required to request user permission.
Like the Android file manager you see in export / import operations.
But asking user to select a folder (and give permission to write) can not happen when traveling.

Unless the app writes in its own folders (data or media).
Although for non experienced users, it would be a bit difficult to find the app folder with a file manager.

Alternatively we could have a management screen for saved screenshots, but this requires more work.

**Reply by 0709wiwiwi** [reply]
Cruiser installed in Android/media ! 
The cx file explorer will comfortably show the image content of an eventual Screenshot_Cruiser folder by the inbuilt cx Imageviewer.
![Screenshots_Cruiser](https://github.com/devemux86/cruiser/assets/54167201/822088b8-ff8a-4a94-aaf9-8d7a3a8caec3)

**Comment by Ciutnik** [comment, marked as answer]
I take a screenshot so often (for OSM changes) that I have configured double-pressing the power button for this purpose. What's more, each screenshot created in this way is assigned a voice note of several seconds. Without it, I wouldn't remember what caught my attention in the field 🙂

This works perfectly. When speaking loudly, I have no problems understanding the audio recording.

**Reply by WolfgangZ42** [reply]
Interesting! Is it a special app? Or an automation profile ... ?

**Reply by Ciutnik** [reply]
This was possible to accomplish with the help of Tasker automation. The advantage is that this works regardless of the application you are currently using.

**Reply by airald** [reply]
Good idea!!! I will add this feature in my automate flow when taking voice notes. (I do something similar to update OSM: I have 10 seconds to take a voice note and is sent by mail with current GPS position and the link to OSM and Google Maps)
